source/core/ExecutionContext.cpp

```diff
@@ -436,7 +436,7 @@ void ExecutionContext::IsrEnqueue(QueueElt* elt)
 DEFINE_VIREO_BEGIN(Execution)
     DEFINE_VIREO_REQUIRE(VirtualInstrument)
     DEFINE_VIREO_FUNCTION(FPSync, "p(i(String))")
-    DEFINE_VIREO_FUNCTION(CurrentBrowserFPS, "p(o(Double) io(" ERROR_CLUST_TYPE_STRING "))")
+    DEFINE_VIREO_FUNCTION(CurrentBrowserFPS, "p(o(Double) io(ErrorClust))")
```

Comment:
Are you asking to rename the type itself in Vireo?
Are you already generating this type by name in DFIR anywhere?